lib/idl/idl_checkers.es6.js

```diff
@@ -320,4 +320,25 @@ Right: ${stringify(ra)})`);
       }
     }
   },
+  function leftAddsNonStandardAPIs(logger, left, right) {
```

Comment:
Could we make this `rightAdds...`? I've been generally treating the left as the "correct, reference" side, and right as the "comparatively different" side.